src/main/java/blusunrize/immersiveengineering/common/blocks/metal/CrusherBlockEntity.java

```diff
@@ -84,6 +88,25 @@ public void tickClient()
 		}
 	}
 
+	@Override
+	public void tickServer() {
+		super.tickServer();
+		if (processQueue.size() > 0 && processQueue.get(0).getRecipe(getLevelNonnull()) != null)
+		{
+			spawnParticles(processQueue.get(0).getRecipe(getLevelNonnull()).input.getItems()[0]);
+		}
+	}
+
+	public void spawnParticles(ItemStack stack)
+	{
+		if (getLevel() instanceof ServerLevel serverLevel)
+		{
+			serverLevel.sendParticles(new ItemParticleOption(ParticleTypes.ITEM, stack), getBlockPos().getX() + (getFacing().getAxis().equals(Axis.X) ? 0.5 : 0), getBlockPos().getY() + 2.125, getBlockPos().getZ() + 0.5 -(getFacing().getAxis().equals(Axis.X) ? 0.5 : 0) , 8, 0, 0, 0, 0.0625);
+			serverLevel.sendParticles(new ItemParticleOption(ParticleTypes.ITEM, stack), getBlockPos().getX() + 0.5, getBlockPos().getY() + 2.125, getBlockPos().getZ() + 0.5, 8, 0, 0, 0, 0.0625);
+			serverLevel.sendParticles(new ItemParticleOption(ParticleTypes.ITEM, stack), getBlockPos().getX() + 1 - (getFacing().getAxis().equals(Axis.X) ? 0.5 : 0), getBlockPos().getY() + 2.125, getBlockPos().getZ() + 0.5 + (getFacing().getAxis().equals(Axis.X) ? 0.5 : 0), 8, 0, 0, 0, 0.0625);
```

Comment:
Can we get some linebreaks in here? You did it for the SampleDrill, I think they'd improve readabilitiy here too.